examples/bungalow-rental-and-usage.yaml

```diff
@@ -0,0 +1,164 @@
+'@context':
+  '@vocab': https://w3id.org/valueflows#
+  xsd: http://www.w3.org/2001/XMLSchema#
+  time: http://www.w3.org/2006/time#
+  time:inXSDDateTimeStamp:
+    '@type': xsd:dateTimeStamp
+  elf: https://elf-pavlik.hackers4peace.net/
+  tashirat: https://tashirat.com/
+  notary: https://notary.example
+
+'@graph':
+
+  - '@id': tashirat:2402b57c-1c45-423a-acb2-f50961c97d65
+    '@type': EconomicResource
+    rdfs:label: Bungalow 6
+
+  - '@id': notary:40bb78fe-28df-41a6-b698-525fa9d411d6
+    '@type': EchangeAgreement
+    skos:note: One year rental agreement of Bungalow 6 for 3000MXN a month
+
+  - '@id': tashirat:953b40f2-4e47-4486-afb2-402512f5269c
+    '@type': Commitment
+    under: notary:40bb78fe-28df-41a6-b698-525fa9d411d6
+    provider: tashirat
+    recipient: elf
+    action: lend
+    involves: tashirat:2402b57c-1c45-423a-acb2-f50961c97d65
+    commitedQuantity:
+      qudt:unit: unit:Unitless
```

Comment:
I wonder if we should investigate using something besides qudt?  I heard of one from Europe I think, that might be simpler, but I'd have to track it down.  Also, I think I found something in qudt for 'each' - maybe Count or Number or something?